### PR TITLE
[doc][yba] YBA install prereqs

### PR DESCRIPTION
**Pull request by ddhodge**
Clarify prerequisites
DOC-240
DOC-279
DOC-251

@netlify /preview/yugabyte-platform/install-yugabyte-platform/install-software/installer/

### CODE REVIEW

**Review by shubin-yb — APPROVED** [review]

File: docs/content/stable/yugabyte-platform/configure-yugabyte-platform/supported-os-and-arch.md [hunk 1]
```diff
@@ -14,6 +14,8 @@ type: docs
 
 Using YugabyteDB Anywhere (YBA), you can deploy YugabyteDB universes on nodes with the following architectures and operating systems.
 
+(For information on prerequisites for the node running YBA, refer to [Prerequisites for YBA](../../install-yugabyte-platform/prerequisites/installer/).)
```
Comment (aishwarya24) [inline docs/content/stable/yugabyte-platform/configure-yugabyte-platform/supported-os-and-arch.md:17]:
```suggestion
-> For information on prerequisites for the node running YBA, refer to [Prerequisites for YBA](../../install-yugabyte-platform/prerequisites/installer/).
```

Comment (aishwarya24) [reply docs/content/stable/yugabyte-platform/configure-yugabyte-platform/supported-os-and-arch.md:17]: how about we make it more prominent?

File: docs/content/preview/yugabyte-platform/install-yugabyte-platform/install-software/installer.md [hunk 2]
```diff
@@ -240,7 +231,7 @@ YugabyteDB Anywhere will end support for Replicated installation at the end of 2
 
 If your YBA installation uses Replicated, you can use YBA Installer to migrate from Replicated.
 
-Review the [prerequisites](#prerequisites). YBA Installer performs the migration in place. Make sure you have enough disk space on your current machine.
+Review the [prerequisites](../../prerequisites/installer/). YBA Installer performs the migration in place. Make sure you have enough disk space on your current machine.
```
Comment (aishwarya24) [inline docs/content/preview/yugabyte-platform/install-yugabyte-platform/install-software/installer.md:234]: Line 231 - Migrating from Replicated using YBA Installer is [Early Access] should be Migrating from Replicated using YBA Installer is **in** [Early Access]

**Review by aishwarya24 — APPROVED** [review]
LGTM. minor comments.

**Review by iSignal — APPROVED** [review]

**Pull request merged**